app/assets/stylesheets/app/dashboard.scss

```diff
@@ -47,36 +43,70 @@
 
   &--head {
     @extend .matrix__user;
+    height: 55px;
     flex: 0 0 65px;
     padding: 0;
-    padding-top: 12px;
   }
 }
 
 .matrix__user-img {
+  display: block;
   max-height: 48px;
   max-width: 48px;
   border-radius: 50%;
   border: 1px solid $secondary-border;
+
+  &--first {
+    @extend .matrix__user-img;
+    border: 3px solid $first-img;
+  }
+
+  &--second {
+    @extend .matrix__user-img;
+    border: 3px solid $second-img;
+  }
+
+  &--third {
+    @extend .matrix__user-img;
+    border: 3px solid $third-img;
+  }
+}
+
+.matrix__user-crown {
+  z-index: 1;
+  height: 48px;
+  width: 48px;
+}
+
+.matrix__user-div {
+  display: inline-block;
+  position: relative;
+  z-index: 1;
+}
+
+.matrix__user-div + .matrix__user-crown {
```

Comment:
Merge rule `.matrix__user-div + .matrix__user-crown` with rule on line 81<br>Selector `matrix__user-div` should be written in lowercase with hyphens<br>Selector `matrix__user-crown` should be written in lowercase with hyphens